sticker-utils/src/bin/sticker-train.rs

```diff
@@ -186,7 +186,7 @@ where
             tagger.validate(seq_lens, tokens, labels)
         };
 
-        let n_tokens = seq_lens.iter().sum::<i32>();
+        let n_tokens = seq_lens.view().iter().sum::<i32>();
```

Comment:
Seems odd/unexpected, that the `NDTensor` doesn't offer the .iter() method. Maybe `.iter()` and `.iter_mut()` can be implemented on `NDTensor`, too?

Reply:
Great idea in general! I guess that we'd also need axis iterators.

Reply:
Added an issue: danieldk/ndarray-tensorflow#2